spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HomeController, type: :controller do
+  describe "#index" do
+    it "renders 'abtion-rails'" do
+      user = FactoryBot.create(:user)
+      request.headers["X-User-Id"] = user.id
+      request.headers["Authorization"] = "Token #{user.authentication_token}"
+
+      get :index, params: { id: user.id }
```

Comment:
This is a horrible amount of setup to need to do.

Reply:
(it seems to be unnecessary?)

Reply:
oh... no authentication needed for the `HomeController`  :O

Reply:
That may or may not be an issue. Either way it should be in a different PR.